### PR TITLE
run bundle: Enable run bundle

### PR DESCRIPTION
**Pull request by jmrodri**
**Description of the change:**
Enable the `run bundle` command. Depends on PR #3861 

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by jmrodri** [comment]
Once the other run bundle PRs merge, this one can get rebased.

File: internal/cmd/operator-sdk/run/cmd.go [hunk 1]
```diff
@@ -34,7 +35,7 @@ Currently only the package manifests format is supported via the 'packagemanifes
 
 	cmd.AddCommand(
 		// TODO(joelanford): enable bundle command when implementation is complete
```
Comment (joelanford) [inline internal/cmd/operator-sdk/run/cmd.go:37]: Remove this comment
```suggestion
```

**Review by joelanford — APPROVED** [review]
One nit. Otherwise
/lgtm

**Review by rashmigottipati — APPROVED** [review]

**Comment by jmrodri** [comment]
Closing because this code was added to the e2e test PR #3831

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Pull request merged**